harness/determined/common/experimental/experiment.py

```diff
@@ -289,3 +303,11 @@ def key(ckpt: checkpoint.Checkpoint) -> Any:
 
     def __repr__(self) -> str:
         return "Experiment(id={})".format(self.id)
+
+    @classmethod
+    def _from_bindings(
```

Comment:
Could we have a quick chat about whether or not to make this constructor public? Maybe if the answer is "it's a bad idea," then talking it through will help give us ideas about a different way to represent the class.

Reply:
I'm not confident enough in our REST APIs to make this public. Our bindings are not user friendly at the moment. I suppose another option is having just one constructor with all arguments, but only ID is required? though that seems confusing to users, who might not know which values are required and when.